test/integ/quote-relay.test.ts

```diff
@@ -0,0 +1,168 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { CurrencyAmount } from '@uniswap/sdk-core';
+import { DAI_MAINNET, ID_TO_NETWORK_NAME, USDC_MAINNET, USDT_MAINNET } from '@uniswap/smart-order-router';
+import { RelayOrder } from '@uniswap/uniswapx-sdk';
+import { UNIVERSAL_ROUTER_ADDRESS } from '@uniswap/universal-router-sdk';
+import { AxiosResponse } from 'axios';
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import chaiSubset from 'chai-subset';
+import _ from 'lodash';
+import { RoutingType } from '../../lib/constants';
+import { QuoteRequestBodyJSON, RoutingConfigJSON } from '../../lib/entities';
+import { QuoteResponseJSON } from '../../lib/handlers/quote/handler';
+import { RelayOrderReactor__factory } from '../../lib/types/ext';
+import { getAmount } from '../utils/tokens';
+import { BaseIntegrationTestSuite, call } from './base.test';
+
+chai.use(chaiAsPromised);
+chai.use(chaiSubset);
+
+const SLIPPAGE = '5';
+
+describe('relayQuote', function () {
+  let baseTest: BaseIntegrationTestSuite;
```

Comment:
maybe we should separate out base test suites of different order types? to reduce setup time and chance of timing out